_includes/newsletter.html

```diff
@@ -4,10 +4,10 @@
             <img src="/assets/img/guides/layout/img-newsletter.svg" alt="newsletter_image" class="img-center"><br>
             <label>Newsletter</label>
             <p>Subscribe to the Gruntwork Newsletter to get notified when updates and new guides are released. No spam. Ever.</p>
-            <form  id="guides-newsletter-form"class="col-sm-10 col-sm-offset-1 col-xs-offset-1 col-xs-10" method="post" action="//gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&amp;id=786bd365d1" target="_blank">
+            <form  id="guides-newsletter-form" class="col-sm-10 col-sm-offset-1 col-xs-offset-1 col-xs-10" method="post" action="//gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&amp;id=786bd365d1" target="_blank">
                 <div class="center">
-                    <input type="email" name="EMAIL" class="email form-control guides-input-box" id="guides-subscribe-email" placeholder="Your Email Address">
-                    <input type="submit" class="btn guides-button" value="Subscribe">
+                    <input type="email" name="EMAIL" minlength="5" class="email form-control guides-input-box" id="guides-subscribe-email" placeholder="Your Email Address" required>
```

Comment:
Not a comment, just more of a question really: Why did you decide to add a minlength and why 5?